src/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/ExternalReferencesTable.cs

```diff
@@ -68,26 +68,17 @@ public bool InitializeCommonFixupsTable(IntPtr moduleHandle)
 
         unsafe public uint GetRvaFromIndex(uint index)
         {
+            throw new PlatformNotSupportedException();
+        }
+
+        unsafe public IntPtr GetIntPtrFromIndex(uint index)
+        {
             Debug.Assert(_moduleHandle != IntPtr.Zero);
 
             if (index >= _RVAsCount)
                 throw new BadImageFormatException();
 
-            return ((uint*)_RVAs)[index];
-        }
-
-        unsafe public IntPtr GetIntPtrFromIndex(uint index)
-        {
-            uint rva = GetRvaFromIndex(index);
-            if ((rva & 0x80000000) != 0)
```

Comment:
Ignore the deletes in this file. This is the throwaway fork.

I'll need to do something with the `GetRvaFromIndex` dependencies on Project N side. I assume we need to get rid of RVAs for non-Windows targets and this will need some factoring changes.